src/tools/perf/perftest.c

```diff
@@ -296,26 +295,28 @@ static void sock_rte_recv(void *rte_group, unsigned src, void *buffer,
     exit(EXIT_FAILURE);
 }
 
-static void rte_report_common(void *rte_group, const ucx_perf_result_t *result,
-                              void *arg, const char *extra_info, int is_final,
-                              int is_multi_thread)
+static void rte_report(void *rte_group, const ucx_perf_result_t *result,
```

Comment:
can we change print_progress to match report_func definition instead of having this wrapper?

Reply:
fixed